demo/admin/src/products/future/generated/ManufacturersGrid.tsx

```diff
@@ -138,6 +138,9 @@ export function ManufacturersGrid() {
             filterable: false,
             sortable: false,
             valueGetter: (params, row) => row.address?.streetNumber,
+            renderCell: ({ value }) => {
+                return typeof value === "number" ? <FormattedNumber value={value} minimumFractionDigits={0} maximumFractionDigits={0} /> : "";
```

Comment:
Using a FormattedNumber for the street number ist strange though...

Consider postal codes: A postal code "5020" would be displayed as "5.020". But it's probably incorrect to use a number for it anyway...

Reply:
so, do you suggest making FormattedNumber optional?

Reply:
Either this, or change the type of postal code to string.

Reply:
even if we change postal code to string (what we should do) - we have to decide of FormattedNumber usage should be optional/configurable

Reply:
IMO it is okay to always use it for `type: "number"`

Reply:
@nsams @johnnyomair I changed `zip` to a string and show it in the future generated grid now, like in the handmade example: 48cc8599e5642c520ef689ca90c45dcb5b471a69